src/main/java/com/github/fabienrenaud/jjb/support/Library.java

```diff
@@ -35,7 +35,9 @@ public enum Library {
     UNDERSCORE_JAVA,
     PUREJSON,
     ANTONS,
-    QUICKBUF_JSON;
+    QUICKBUF_JSON,
+    WAST
+    ;
```

Comment:
Move semi-colon to the same line as WAST.

Reply:
Okay, it has been adjusted